crates/fluvio-controlplane-metadata/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-controlplane-metadata"
 edition = "2021"
-version = "0.28.2"
+version = "0.28.3"
```

Comment:
this doesn't need to be bump up since `0.28.2` is not released

Reply:
Make sense! I thought that It was based from master version. Fixed